misc/test/examples_test.go

```diff
@@ -148,6 +148,53 @@ func TestExamples(t *testing.T) {
 				})
 			},
 		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "cloud-js-containers"),
+			SkipBuild: true,
+			Config: map[string]string{
+				// Use us-west-2 to assure fargate is availablecd
+				"aws:region":           "us-west-2",
+				"cloud-aws:useFargate": "true",
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["hostname"], func(body string) bool {
+					return assert.Contains(t, body, "<p>Hello, containers!</p>")
+				})
+			},
+		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "cloud-js-httpserver"),
+			SkipBuild: true,
+			Config: map[string]string{
+				"cloud:provider": "aws",
+				"aws:region":     awsRegion,
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["endpoint"].(string)+"/hello", func(body string) bool {
+					return assert.Contains(t, body, "{\"route\":\"/hello\",\"count\":1}")
+				})
+			},
+		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:           path.Join(cwd, "..", "..", "cloud-ts-url-shortener-cache-http"),
+			Verbose:       true,
+			DebugLogLevel: 8,
+			DebugUpdates:  true,
+			SkipBuild:     true,
+			Config: map[string]string{
+				"url-shortener-cache:redisPassword": "s3cr7Password",
+				"cloud:provider":                    "aws",
+				// Use us-west-2 to assure fargate availability
+				"aws:region":                     "us-west-2",
```

Comment:
Hmm, ok, I though I had issues but us-west but i agree it would be better to just use the variable that all the other tests are using.  i'll try changing.

Reply:
Oh - are we running these examples in `us-west`?  That region isn’t generally fully up to date - should probably run in Oregon, Ohio, Virginia or Dublin just so we don’t run into regional service availability issues in the future.